chaincfg/params.go

```diff
@@ -476,6 +492,10 @@ var SimNetParams = Params{
 	// Mempool parameters
 	RelayNonStdTxs: true,
 
+	// Human-readable part for Bech32 encoded segwit addresses, as defined in
+	// BIP 173.
+	Bech32HRPSegwit: "tb", // always tb for test net
```

Comment:
Can we make this "sb" (and adjust the comments accordingly)?  We don't want simnet to be able to use testnet addresses.  Hence why the p2pkh addrs start with S (just below).